rust/crd/src/resource.rs

```diff
@@ -0,0 +1,277 @@
+use crate::storage;
+use lazy_static::lazy_static;
+use snafu::Snafu;
+use stackable_operator::{
+    commons::resources::{CpuLimits, MemoryLimits, NoRuntimeLimits, Resources},
+    config::merge::Merge,
+    k8s_openapi::{
+        api::core::v1::ResourceRequirements, apimachinery::pkg::api::resource::Quantity,
+    },
+};
+use strum::{EnumDiscriminants, IntoStaticStr};
+
+#[derive(Debug, Clone, PartialEq)]
+pub enum RoleResourceEnum {
```

Comment:
Any particular reason for the Enum suffix? Reminds me of the Hungarian notation ^^

Reply:
It's a leftover from trying different things out. I will rename it. Also I liked the Hungarian notation :)